### ISSUE TITLE
Can't build today - claw issue?

### ISSUE THREAD

**Issue by quicklisp** [borodust/bodge-nanovg#2]
http://report.quicklisp.org/2020-02-14/failure-report/bodge-nanovg.html

I get this: ```
Error while parsing arguments to C-INCLUDE DEFMACRO:unknown keyword: :INCLUDES; expected one of :IN-PACKAGE, :INCLUDE-SOURCES, :INCLUDE-DEFINITIONS, :EXCLUDE-SOURCES, :EXCLUDE-DEFINITIONS, :RENAME-SYMBOLS, :SYSINCLUDES, :WINDOWS-ENVIRONMENT```

**Comment by borodust** [comment]
Yeah. Version mismatch most likely. `bodge` collection got quite far ahead of quicklisp versions and when fixing `stable` issue i might have pushed too new or too old version of `bodge-nanovg`. Hopefully, I'll sort it out this weekend.
